tools/bridge/docker/docker-compose-nodes-development.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.4'
+services:
+
+  node_foreign:
+    image: parity/parity:v2.5.5-stable
+    command:
+      - "--light"
+      - "--no-download"
+      - "--auto-update=none"
+      - "--chain=ropsten"
+      - "--base-path=/data"
+      - "--no-hardware-wallets"
+      - "--jsonrpc-interface=all"
+      - "--jsonrpc-apis=safe"
+      - "--jsonrpc-hosts=all"
+      - "--jsonrpc-cors=all"
+      - "--no-ipc"
+      - "--no-secretstore"
+      - "--no-color"
+    volumes: ['./data/ropsten:/data']
+    restart: unless-stopped
+    mem_limit: 2G
+    mem_reservation: 16M
+    stop_grace_period: 1m
+
+  node_home:
+    image: trustlines/tlbc-testnet:release4044
+    command:
+      - "--parity-args"
+      - "--light"
+      - "--no-hardware-wallets"
+      - "--jsonrpc-apis=safe"
+      - "--jsonrpc-hosts=all"
+      - "--jsonrpc-cors=all"
+      - "--no-ipc"
+      - "--no-secretstore"
+    volumes: ['./data/laika:/home/parity/.local/share/io.parity.ethereum/chains_light']
+    restart: unless-stopped
```

Comment:
Suggestion: Possibly you want to use `yaml` anchor to avoid repeating of the common variables and overwrite only the few differences? :thinking:

Reply:
I'd rather have it more explicit and readable than short unless there's a need to shorten it.